asn1ate/parser.py

```diff
@@ -246,7 +246,8 @@ def annotation(t):
     defined_type = Optional(module_reference + Suppress('.'), default=None) + typereference + Optional(size_constraint, default=None)
 
     # TODO: consider exception syntax from 24.1
-    extension_marker = Unique(ELLIPSIS)
+    # At the moment we do not do anything with ELLIPSIS - so suppress it
+    extension_marker = Suppress(Unique(ELLIPSIS))
```

Comment:
Was this necessary for something in this patch? If not, I'd prefer it if the extension marker was still included for now, so we can dig it out of sema in codegen.